include/QueueWithArray.hpp

```diff
@@ -0,0 +1,92 @@
+#include<iostream>
+#include"Queue.hpp"
+
+template <class T>
+class QueueWithArray : public Queue<T>
+{
+private:
+  T* queue;
+  int front;
+  int rear;
+  int size;
+  int capacity;
```

Comment:
Please follow the variable naming convention planned to use for this project. Read under CONTRIBUTION.md to know about convention used.

For private variables,
variables should start with the _
Eg: 
_queue
_rear